app/models/user.rb

```diff
@@ -19,6 +21,18 @@ def self.new_token
     SecureRandom.urlsafe_base64
   end
 
+  def self.create_anonimous
+    anonimous_token = SecureRandom.hex(8)
+    create username: anonimous_token, email: "#{anonimous_token}@stub",
```

Comment:
Looks like it's better to get a "validation failed" from `create!` method here, than swallow an error with `create` and get a cryptic "method not found" somewhere down the flow.